zerver/lib/management.py

```diff
@@ -0,0 +1,72 @@
+# Library code for use in management commands
+from __future__ import absolute_import
+from __future__ import print_function
+
+from argparse import ArgumentParser
+from django.core.exceptions import MultipleObjectsReturned
+from django.core.management.base import BaseCommand, CommandError
+from typing import Any, Dict, Optional, Text
+
+from zerver.models import get_realm, Realm, UserProfile
+
+def is_integer_string(val):
+    # type: (str) -> bool
+    try:
+        int(val)
+        return True
+    except ValueError:
+        return False
+
+class ZulipBaseCommand(BaseCommand):
+    def add_realm_args(self, parser, required=False):
+        # type: (ArgumentParser, bool) -> None
+        parser.add_argument(
+            '-r', '--realm',
+            dest='realm_id',
+            required=required,
+            type=str,
+            help='The numeric or string ID (subdomain) of the Zulip organization to modify.')
+
+    def get_realm(self, options):
+        # type: (Dict[str, Any]) -> Optional[Realm]
+        val = options["realm_id"]
+        if val is None:
+            return None
+
+        # If they specified a realm argument, we need to ensure the
+        # realm exists.  We allow two formats: the numeric ID for the
+        # realm and the string ID of the realm.
+        if is_integer_string(val):
+            try:
+                realm = Realm.objects.get(id=val)
+            except Realm.DoesNotExist:
+                realm = None
+        else:
+            realm = get_realm(val)
+
+        if realm is None:
+            raise CommandError("The realm '%s' does not exist. Aborting." %
```

Comment:
Can we have message like this?
`
The realm with the id '%s' does not exist. Aborting.
`

I think it sounds better if the user passed an integer id and the realm with that id does not exist.